src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -103,6 +103,15 @@ export class HuiThermostatCard extends hassLocalizeLitMixin(LitElement)
       return html``;
     }
     const stateObj = this.hass.states[this._config.entity] as ClimateEntity;
+    if (!stateObj) {
+      return html`
+        <ha-card>
+          <div class="not-found">
```

Comment:
The styling will not be available, missing `${this.renderStyle()}`

Reply:
Oh

Reply:
https://github.com/home-assistant/home-assistant-polymer/pull/2538